tbtadm/controller.cpp

```diff
@@ -308,6 +319,30 @@ void tbtadm::Controller::devices()
     }
 }
 
+void tbtadm::Controller::peers()
+{
+    for (auto dir : boost::make_iterator_range(
+             fs::directory_iterator(sysfsDevicesPath), {}))
+    {
+        if (dir.status().type() != fs::directory_file)
+        {
+            continue;
+        }
+        if (!isXDomain(dir.path()))
+        {
+            continue;
+        }
+
+        chdir(dir.path());
+
+        // TODO: better formatting
+        const auto routeString = dir.path().filename().string();
+
+        m_out << routeString << '\t' << readVendor(vendorFilename) << '\t'
+              << readDevice(deviceFilename) << std::endl;
```

Comment:
Maybe we want more details here? E.g. the services it supports? (But this can be added later if decided.)

Reply:
Yup, I think we can add those details later as needed :)